core/src/Content.cpp

```diff
@@ -81,15 +81,15 @@ void Content::erase(int pos, int n)
 	bytes.erase(bytes.begin() + pos, bytes.begin() + pos + n);
 	for (auto& e : encodings)
 		if (e.pos > pos)
-			pos -= n;
+			e.pos -= n;
```

Comment:
Wow, that is stunning. 3 years and this has not surfaced, yet. Thanks for fixing this total fail.